concepts/type-conversion/about.md

```diff
@@ -1,3 +1,43 @@
-# About
+# About Type Conversion
 
-TODO: add information on type-conversion concept
+In Go, assignment of a value between different types requires explicit conversion.
+For example, this is a compiler error:
+
+```go
+var x int = 42
+var f float64 = x // compiler error x is an int not a float64
+```
+
+Converting between types is done via a function with the name of the type to convert to.
+A value `v` is converted to type `T` using `T(v)`.
+For example, to convert an `int` to a `float64` you would need to do the following:
+
+```go
+var x int = 42 // x has type int
+f := float64(x) // f has type float64 (ie. 42.0)
+```
+
+## Converting between primitive types and strings
+
+There is a `strconv` package for converting between primitive types (like `int`) and `string`.
```

Comment:
As introduction to `strconv` I would use the following pitfall so the student is warned about that
```go
var num int = 65
str := string(num)
// str is now "A" not "65"
```
Probably not needed for the introduction but definitely good content for the about imo.